examples/test_feature_example_group_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
+
+describe "test feature example group" do
+  feature "test.feature"
```

Comment:
The main difference is you get `before` blocks, `lets`, and rails example groups, etc...